README.md

```diff
@@ -59,6 +59,18 @@ To make sure that the Ruby LSP can find the version manager scripts, make sure t
 configuration script (e.g.: ~/.bashrc, ~/.zshrc) and that the SHELL environment variable is set and pointing to the
 default shell.
 
+#### Configuring a formatter
+
+The tool to be used for formatting files can be configured with the following setting.
+
+```jsonc
+// Available options
+//    auto: automatically detect the formatter based on the app's bundle
```

Comment:
Do we want to mention `auto` is the default like we did in the `Ruby version managers` section?

```suggestion
//    auto: automatically detect the formatter based on the app's bundle (default)
```